pkgs/by-name/mu/multimidicast/package.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, alsa-lib }:
+stdenv.mkDerivation rec {
+  name = "multimidicast";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "https://llg.cubic.org/tools/multimidicast/multimidicast-${version}.tar.gz";
+    hash = "sha256-NmDPQS+Rib1ro20PrhDYHX7kAvhmENlPKSptljA8vh4=";
+  };
```

could be my personal preference, but I prefer `fetchzip` over `fetchurl`, as it unpacks the tarball itself and lets one inspect the source directory with just `nix-build -A PACKAGE.src` (or `nix build FLAKE#PACKAGE.src`)
```suggestion
  src = fetchzip {
    url = "https://llg.cubic.org/tools/multimidicast/multimidicast-${version}.tar.gz";
    hash = "";
  };
```
hash will change, copy it from the build error